css-brochure-form/styles.css

```diff
@@ -0,0 +1,81 @@
+.container {
+  width: 375px;
+  margin: 0 auto;
+}
+
+h2 {
+  color: #ec5d57;
+  font-family: 'Brandon Grotesque', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  text-transform: uppercase;
+  text-align: center;
+  margin-bottom: 0;
+}
+
+label {
+  color: black;
+  font-family: 'Brandon Grotesque', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 11px;
+  text-transform: uppercase;
+  display: block;
+  padding-top: 10px;
+}
+
+input {
+  border-radius: 5px;
+  border: 0.5px solid #a6aaa9;
+  width: 100%;
+  padding-top: 8px;
+  padding-bottom: 8px;
+}
+
+select {
+  background-color: #f3f3f3;
+  border-radius: 3px;
+  border: 0.5px solid #a6aaa9;
+  width: 100%;
+  padding-top: 8px;
+  padding-bottom: 8px;
+}
+
+button {
+  color: white;
+  background-color: #ec5d57;
+  font-family: 'Brandon Grotesque', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  text-transform: uppercase;
+  border: 0;
+  border-radius: 15px;
+  padding-right: 15px;
+  padding-left: 15px;
+  padding-top: 5px;
+  padding-bottom: 5px;
+  margin: 10px;
+  float: right;
```

Comment:
Generally, you want to avoid using `float` in situations like this. What I would have done, is put the `<button>` inside of a `<div>` (or another block level element), and give the `<div>` `text-align: right` to push it's children to the end of the block's container

Reply:
ah dang i tried <text-align> first but i didn't do <div> so it didn't work! thank you!!